keyboards/willoucom/keypad/info.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
Can you please run `qmk format-json` on this file.

Reply:
done